xchange-okex/src/main/java/org/knowm/xchange/okex/v5/OkexAdapters.java

```diff
@@ -129,24 +131,22 @@ public static OkexOrderRequest adaptOrder(LimitOrder order) {
         .build();
   }
 
-  public static OrderBook adaptOrderBook(
-      OkexResponse<List<OkexOrderbook>> okexOrderbook, Instrument instrument) {
+  public static LimitOrder adaptLimitOrder(OkexPublicOrder okexPublicOrder, Instrument instrument, OrderType orderType) {
+    return adaptOrderbookOrder(okexPublicOrder.getVolume(), okexPublicOrder.getPrice(), instrument, orderType);
+  }
 
+  public static OrderBook adaptOrderBook(List<OkexOrderbook> okexOrderbooks, Instrument instrument) {
     List<LimitOrder> asks = new ArrayList<>();
     List<LimitOrder> bids = new ArrayList<>();
 
-    okexOrderbook
-        .getData()
+    okexOrderbooks
         .get(0)
         .getAsks()
         .forEach(
             okexAsk ->
-                asks.add(
-                    adaptOrderbookOrder(
-                        okexAsk.getVolume(), okexAsk.getPrice(), instrument, OrderType.ASK)));
+                asks.add(adaptLimitOrder(okexAsk, instrument, OrderType.ASK)));
 
-    okexOrderbook
-        .getData()
+    okexOrderbooks
```

Comment:
why did this side of the book not change to use `adaptLimitOrder`

Reply:
Fixed. Sorry about this mistake.